CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
-## Unreleased
+## 23.13.0
 
 * Update documentation for the meta tags component ([PR #1870](https://github.com/alphagov/govuk_publishing_components/pull/1870))
```

Comment:
If people would look back at this might question why a documentation update was considered a minor release (or a release at all) – could we update this entry to capture the attribute addition too?